src/content/en/updates/2018/12/get-installed-related-apps.md

```diff
@@ -115,15 +106,20 @@ Check out the [`getInstalledRelatedApps()` API Demo][demo] and
 If you want to experiment with the API locally, without an origin trial,
 enable the `#enable-experimental-web-platform-features` flag in `chrome://flags`.
 
-### Establish a relationship between your apps {: #relationship }
+## See it in action
+
+1. Using Chrome 79 or later on Android, open the [`getInstalledRelatedApps()` demo][demo].
+2. Install the demo app from the Play store and refresh the [demo][demo] page.
```

Comment:
Is there no way to link to the app on the play store? What do web sites do when they say 'get our app'.